bower.json

```diff
@@ -20,7 +20,8 @@
     "iron-flex-layout": "^1.0.0",
     "cosmoz-i18next": "neovici/cosmoz-i18next#master",
     "cosmoz-dialog": "neovici/cosmoz-dialog#master",
-    "iron-list": "PolymerElements/iron-list#^2.0.4"
+    "iron-list": "PolymerElements/iron-list#^2.0.4",
+    "cosmoz-treenode": "Neovici/cosmoz-treenode"
```

Comment:
Yes, the navigator gets a `Cosmoz.Tree` from outside the element.
It can be a `Cosmoz.DefaultTree` or some other implementation.
`Cosmoz.Tree` acts as an "abstract class" and defines methods that the implementations need to implement.

Yes the last part is wrong, we expect the tree to be created for us.

Also, it's a weird place to comment about this. This change is to import the "cosmoz-treenode" element which we haven't really touched during this PR, so I don't see why the dependency was added.
Ah, now I realize! :) It's just a typo, it should depend on `Neovici/cosmoz-tree#master` rather than `Neovici/cosmoz-treenode` :)